### PR TITLE
add Queue desc

### PR DESCRIPTION
**Pull request by alwaysdark**
add Queue desc

### CODE REVIEW

File: pkg/apis/scheduling/v1alpha1/type.go [hunk 1]
```diff
@@ -33,14 +33,25 @@ type Queue struct {
...
+	PriorityClassName   string         `json:"priorityClassName,omitempty" protobuf:"bytes,3,opt,name=priorityClassName"`
 }
 
 // QueueStatus defines the observed state of Queue
 type QueueStatus struct {
 	// TODO
 }
 
+// +k8s:openapi-gen=true
+// QueuePolicy describes how to scheduling multi queue
```
Comment (denkensk) [inline pkg/apis/scheduling/v1alpha1/type.go]: QueuePolicy defines the queueing policy for the elements in the queue

**Review by denkensk — COMMENTED** [review]
Update a comment

File: pkg/apis/scheduling/v1alpha1/type.go (hunk 1 above)
Comment (alwaysdark) [reply pkg/apis/scheduling/v1alpha1/type.go]: ok

**Comment by denkensk** [comment]
/lgtm
/approve

Thanks @alwaysdark

**Pull request merged**